src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,21 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>{' Posted by  '}</p>
+      <UserInfo user={post.user} />
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+    {post?.comments.length > 0 ? (
+      <CommentList comments={post.comments} />
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    )}
```

Comment:
Yep, that is better.
1. Let's add an empty line after `<p>` element
2. What is `post?.comments`, this is not a typescript yet😉
3. When there is no comments you should render
```
<>
  <hr />
  <b data-cy="NoCommentsMessage">No comments yet</b>
</>

Reply:
1. Added
2. Alexandra used it to check if the variable is an object
3. Missed the hr, thanks!